packages/datadog-instrumentations/src/helpers/hook.js

```diff
@@ -41,6 +34,14 @@ function Hook (modules, hookOptions, onrequire) {
     if (moduleExports && moduleExports.default) {
       moduleExports.default = safeHook(moduleExports.default, moduleName, moduleBaseDir)
     }
+
+    const parts = [moduleBaseDir, moduleName].filter(Boolean)
+    const filename = path.join(...parts)
+
+    if (this._patched[filename]) return moduleExports
+
+    this._patched[filename] = true
+
```

Comment:
You should leave this chunk in safeHook, and just not do the `moduleExports.default` check here, and instead do it inside safeHook, like:

```js
if (moduleExports && moduleExports.default) {
  moduleExports.default = onrequire(moduleExports.default, moduleName, moduleBaseDir, moduleVersion)
}
return onrequire(moduleExports, moduleName, moduleBaseDir, moduleVersion)
```